http_check/tests/test_utils.py

```diff
@@ -0,0 +1,41 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+import sys
+
+import pytest
+import mock
+
+from datadog_checks.dev import temp_dir
+from datadog_checks.http_check.utils import get_ca_certs_path, _get_ca_certs_paths
+
+
+@pytest.mark.unit
+def test_get_ca_certs_path():
+    with pytest.raises(OSError):
+        get_ca_certs_path()
+
+
+@pytest.mark.unit
+def test__get_ca_certs_paths():
```

Comment:
Can we add the case where a ca_cert is provided in the init_config?

Reply:
Added in `test__init__`